R/reportRenovation.R

```diff
@@ -0,0 +1,105 @@
+              silent = silent),
+    reportAgg(v_renovation,
+              "Renovation|Residential (bn m2/yr)", brickSets,
+              agg = c(bs = "all", hs = "all", bsr = "all0", hsr = "all", vin = "all", loc = "all", typ = "res", inc = "all"),
+              silent = silent),
+    reportAgg(v_renovation,
+              "Renovation|Commercial (bn m2/yr)", brickSets,
+              agg = c(bs = "all", hs = "all", bsr = "all0", hsr = "all", vin = "all", loc = "all", typ = "com", inc = "all"),
+              silent = silent),
+
+
+    ## by building type ====
+    reportAgg(v_renovation,
+              "Renovation|Residential|{typ} (bn m2/yr)", brickSets,
+              agg = c(bs = "all", hs = "all", bsr = "all0", hsr = "all", vin = "all", loc = "all", inc = "all"),
+              rprt = c(typ = "res"),
+              silent = silent),
+
+
+    ## by location ====
+    reportAgg(v_renovation,
+              "Renovation|Residential|{loc} (bn m2/yr)", brickSets,
+              agg = c(bs = "all", hs = "all", bsr = "all0", hsr = "all", vin = "all", typ = "res", inc = "all"),
+              rprt = c(loc = "all"),
+              silent = silent),
+
+
+    ## by initial heating system ====
+    reportAgg(v_renovation,
+              "Renovation|Residential|Initial|{hs} (bn m2/yr)", brickSets,
+              agg = c(bs = "all", bsr = "all0", hsr = "all", vin = "all", loc = "all", typ = "res", inc = "all"),
+              rprt = c(hs = "all"),
+              silent = silent),
+
+
+    ## by building type + initial heating system ====
+    reportAgg(v_renovation,
+              "Renovation|Residential|{typ}|Initial|{hs} (bn m2/yr)", brickSets,
+              agg = c(bs = "all", bsr = "all0", hsr = "all", vin = "all", loc = "all", inc = "all"),
+              rprt = c(hs = "all", typ = "res"),
+              silent = silent),
+
+
+    ## by final heating system ====
+    reportAgg(v_renovation,
+              "Renovation|Residential|Final|{hsr} (bn m2/yr)", brickSets,
+              agg = c(bs = "all", hs = "all", bsr = "all0", vin = "all", loc = "all", typ = "res", inc = "all"),
+              rprt = c(hsr = "all"),
+              silent = silent),
+
+
+    ## by final heating system, without zero state ====
+    reportAgg(v_renovation,
+              "Renovation|Residential|Final with zero|{hsr} (bn m2/yr)", brickSets,
+              agg = c(bs = "all", hs = "all", bsr = "all0", vin = "all", loc = "all", typ = "res", inc = "all"),
+              rprt = c(hsr = "all0"),
+              silent = silent),
+
+
+    ## by initial and final heating system ====
+    reportAgg(v_renovation,
+              "Renovation|Residential|{hs}|{hsr} (bn m2/yr)", brickSets,
+              agg = c(bs = "all", bsr = "all0", vin = "all", loc = "all", typ = "res", inc = "all"),
+              rprt = c(hs = "all", hsr = "all0"),
+              silent = silent)
```

Comment:
Not sure we really want to report full renovation transitions. If we get into the stock subsets here, this let's the number of variables explode.

Reply:
I agree, I'm also not sure about this. I thought this might be useful at some point, but given that it is difficult to visualize this level of complexity anyhow, we can also drop it. Besides, as soon as we have the identical replacement variable, some more information on which hs goes into which becomes available, so indeed we might not need this.